### DISCUSSION TITLE
How does ALE display lint message in command terminal on line hover?

### DISCUSSION THREAD

**Discussion by rkshthrmsh** [w0rp/ale#4395] [Q&A - Ask for help with problems]
I am trying to understand how ALE displays the lint message in command temrinal when the cursor hovers over the line? Requesting for two reasons:

1. I would like to see if there is a way to copy the message directly to clipboard for reusing elsewhere, such as a Google search.
2. I am attempting to use a similar method to print LSP messages to the command line, so that both ALE and LSP message display can be in the same position. Currently LSP diagnostic messages require using a floating window, which is not always preferable.

**Comment by w0rp** [comment]
`:ALEDetail` will show the detailed message in a preview window by default, which you can copy lines from. You can also read `:mes` to get lines from that.
